.github/workflows/python-package.yml

```diff
@@ -52,6 +52,8 @@ jobs:
           TIMEPLUS_API_KEY: ${{ secrets.TIMEPLUS_API_KEY }}
           TIMEPLUS_HOST: ${{ secrets.TIMEPLUS_HOST }}
           TIMEPLUS_WORKSPACE: ${{ secrets.TIMEPLUS_WORKSPACE }}
+          EXTERNAL_STREAM_PASSWORD: ${{ secrets.EXTERNAL_STREAM_PASSWORD }}
+          TIMEPLUS_REPLICATION_NUMBER: 3
```

Comment:
`TIMEPLUSD_REPLICATION_NUMBER` ?

Reply:
I think it is OK, as the other component wont do replication